lib/metasploit/model/login/status.rb

```diff
@@ -36,6 +36,9 @@ module Metasploit::Model::Login::Status
   # When `Metasploit::Credential::Login#service` reports that `Metasploit::Credential::Login#core` are not correct.
   INCORRECT = 'Incorrect'
 
+  # When `Metasploit::Credential::Login#service` reports that `Metasploit::Credential::Login#core` has a public part that is verified as being incorrect.
+  INCORRECT_PUBLIC_PART = 'Incorrect Public Part'
```

Comment:
There's a comment at the top that suggests the following naming convention for new strings in this class:

> The String should work in the sentences 'Login status is <status>' and 'Login is <status>'.

So in this case it doesn't quite work:
- `Login status is Incorrect Public Part`
- `Login is incorrect Public Part`

In terms of english, potentially `Not Found` would work better here?
- `Login status is Not found`
- `Login is Not Found`

But I'm not sure if that's any better or conveys the same intent/clarity to the user of the API. Open to suggestions.

Reply:
I think it's probably fine considering that IMO it makes as much sense as `Login status is Incorrect` which is already not great

Reply:
Hm. I think we'd benefit from a few more constants here for more granular reporting; but I think this is definitely the first step to fixing an obvious piece of missing functionality 😄 

If there's no strong opinions we could go with INCORRECT_PUBLIC_PART for now as part of patch release

---

Longer term: In the future if we decide to redesign the constants we can release it as a major gem bump. I'm thinking we'd need more cycles in framework/pro to investigate the impact of introducing constants for invalid versus correct, and whether a new contributing user would appreciate the differences when contributing new scanners.

Another edgecase I ran into was, Kerberos can report `KDC_ERR_CLIENT_REVOKED` - which is a very generic error:
> This might be because of an explicit disabling or because of other restrictions in place on the account. For example: **account disabled, expired, or locked out**

It's not clear in the current constants if this should map to Metasploit's single constant of `DENIED_ACCESS`, `LOCKED_OUT`, or `DISABLED`, and we don't have a constant for `EXPIRED`.

Arguably login status would support union functionality for the most flexibility here, but that's definitely all out of scope for now 😄

Reply:
I'm happy to rename this to `INVALID_PUBLIC_PART` if that's preferred, I was initially just going for consistency with `INCORRECT`

Reply:
Let's go with `INVALID_PUBLIC_PART` for now.

There has been some discussion about a `login` validity periods that may inform how to address the concept of `EXPIRED`. I definitely see that as scope creep on the needs here though.

On the `KDC_ERR_CLIENT_REVOKED` part for `Kerberos` specific usage I think `DENIED_ACCESS` may be the place to start and future iterations can address how to enumerate more specificity into status values.